### DISCUSSION TITLE
Support "::backdrop"

### DISCUSSION THREAD

**Discussion by tyler36** [tailwindlabs/tailwindcss#8082] [Ideas]
## Request

Add the ability to target the `dialog::backdrop`  with TailwindCSS background colors.

Instead of:
```html
<style>
    dialog::backdrop {
        background-color: rgba(255, 0, 0, 0.5);
    }
</style>

<dialog open>
     hello world
</dialog>
```

I'd like to:

```html
<dialog open class="backdrop-red-500 backdrop-opactiy-50">
     hello world
</dialog>
```

I understand that it is a relatively new element, but it appears to have [good modern browser support](https://caniuse.com/?search=dialog). 

## Inspiration
I just finished the Laracast [Simpler Modals With the Dialog Element](https://laracasts.com/series/jeffreys-larabits/episodes/23) episode where he demo's the `dialog` attribute.

I was unable to find anything in the docs (or issues) to show support for `dialog::backdrop`

**Comment by adamwathan** [comment]
Will be in the next release! 🥳

https://github.com/tailwindlabs/tailwindcss/pull/7924

**Comment by tyler36** [comment]
Thank you for the quick reply. I forgot to check the PRs. 😢 

Congrats on upcoming release and thank you for the modern take on styling!
